### PR TITLE
Add ability to define Enterprise tag with title for nested sections

### PR DESCRIPTION
**Pull request by serg**
/assign @scheeles

### CODE REVIEW

File: layouts/partials/header.html [hunk 1]
```diff
@@ -75,12 +75,14 @@
...
-              {{ end }}
-              {{.Title}}
-            </h1>
+            <div{{with .Params.enterprise}} class="title-holder"{{end}}>
+              <h1>
+                {{ if eq .Kind "taxonomy" }}
+                  {{.Kind}} ::
+                {{ end }}
+                {{.Title}}
+              </h1>{{with .Params.enterprise}}<span class="title-holder-tag">Enterprise Suite</span>{{end}}
```
Comment (xrstf) [inline layouts/partials/header.html:84]: It's called `Enterprise Edition` :-)

File: layouts/partials/header.html [hunk 2]
```diff
@@ -84,12 +84,14 @@
...
-              {{ end }}
-              {{.Title}}
-            </h1>
+            <div{{with .Params.enterprise}} class="title-holder"{{end}}>
+              <h1>
+                {{ if eq .Kind "taxonomy" }}
+                  {{.Kind}} ::
+                {{ end }}
+                {{.Title}}
+              </h1>{{with .Params.enterprise}}<span class="title-holder-tag">Enterprise Edition</span>{{end}}
```
Comment (serg) [inline layouts/partials/header.html:93]: Now fixed

**Pull request merged**